doc/classes/@GlobalScope.xml

```diff
@@ -2596,6 +2596,8 @@
 		</constant>
 		<constant name="PROPERTY_USAGE_ARRAY" value="1073741824" enum="PropertyUsageFlags">
 		</constant>
+		<constant name="PROPERTY_USAGE_SHORTCUT" value="-2147483648" enum="PropertyUsageFlags">
```

Comment:
It was using `uint32_t` to store the values, but I guess enum itself is interpreted as signed (since Variant is casting it to `int64_t`), changed it to `int64_t`. But doc page did not change, there seems to be something else in the doc generator or Variant that still casting it to 32-bit.